docs/gc.md

```diff
@@ -55,28 +55,28 @@ The following table shows the heap configuration and the GC cycles and operation
 
 | Policy         | Heap configuration |  GC cycles / operations |
 |----------------|--------------------|-------------------------|
-|`gencon`        | Two areas: *nursery* and *tenure* <br> Two generation groups: new/older |Global GC cycle: concurrent *mark-sweep* operations, optionally followed by a *compact* operation <br> Partial GC cycle: STW *scavenge* operation or concurrent *scavenge* operation (if `-Xgc:concurrentScavenge` is set) |
-|`balanced`      | Multiple regions of equal size <br> Multiple generations: 24 |Global GC cycle: incremental concurrent *mark* operation (*global mark phase*), followed by a global STW *sweep* operation <br> Partial GC cycle: STW *copy forward* operation  or STW *mark-compact* operation |
-|`optavgpause`   | Single area: *flat* | Global GC cycle: concurrent *mark-sweep* operations, optionally followed by a *compact* operation |
-|`optthruput`    | Single area: *flat*  | Global GC cycle: STW *mark-sweep* operations, optionally followed by a *compact* operation|
-|`metronome`     | Multiple regions <br> Segregated by size class | Global GC cycle: incremental STW *mark-sweep* operation in small interruptible steps |
-|` nogc`         | Single area: *flat* | No GC cycles |
+|`gencon`        | Two areas: *nursery* and *tenure* <br> Two generation groups: new/older |Global GC cycle: concurrent *mark-sweep* operations, optionally followed by a *compact* operation <br> Partial GC cycle: STW *scavenge* operation or concurrent *scavenge* operation (if optionally enabled) |
+|`balanced`      | Multiple regions of equal size <br> Multiple generations |Global GC *mark* cycle: incremental concurrent *mark* operation (*global mark phase*) <br> Partial GC cycle: STW *copy forward* operation and optional *mark*, *sweep* or *compact* operations |
+|`optavgpause`   | One area: *flat* <br> One generation | Global GC cycle: concurrent *mark-sweep* operations, optionally followed by a *compact* operation |
```

Comment:
I am not sure how to improve this without overloading by facts (may be it is acceptable "as is" as outline). 
In reality there are two types of GC cycles on optavgpause:
- Concurrent Mark/STW phase (Final Mark + Partial Sweep)/Concurrent Sweep
- Concurrent Mark/STW phase (Final Mark, Sweep, Compact)

Reply:
Yes, I've had similar discussions with Aleks. We're keeping this table relatively high-level and need to elaborate in the detailed sections. Not sure it is all there yet, but a work in progress. However, I'm keen to get as much content is for 0.24.0 as possible and iterate on it afterwards.